INFO: 2023-08-21 22:55:00 Argument batch_size: 8
INFO: 2023-08-21 22:55:00 Argument code_test: True
INFO: 2023-08-21 22:55:00 Argument cut_off: True
INFO: 2023-08-21 22:55:00 Argument data_train_path: ./../data/data_96/train/
INFO: 2023-08-21 22:55:00 Argument data_val_path: ./../data/data_96/val/data_val_chr19_96.npy
INFO: 2023-08-21 22:55:00 Argument device_number: 1
INFO: 2023-08-21 22:55:00 Argument early_stoppage_epochs: 10
INFO: 2023-08-21 22:55:00 Argument epoch: 1
INFO: 2023-08-21 22:55:00 Argument local_rank: 0
INFO: 2023-08-21 22:55:00 Argument loss: single_channel_MSE_no_vgg
INFO: 2023-08-21 22:55:00 Argument max_HiC: 100
INFO: 2023-08-21 22:55:00 Argument num_gpu: 1
INFO: 2023-08-21 22:55:00 Argument num_workers: 1
INFO: 2023-08-21 22:55:00 Argument patch_size: 96
INFO: 2023-08-21 22:55:00 Argument resume_epoch: 0
INFO: 2023-08-21 22:55:00 Argument resume_path: None
INFO: 2023-08-21 22:55:00 Argument rgb: False
INFO: 2023-08-21 22:55:00 Argument train_dataset: hic
INFO: 2023-08-21 22:55:00 Argument val_datasets: ['hic']
INFO: Traceback:
INFO: Traceback (most recent call last):
  File "./train_1d.py", line 342, in <module>
    train(model, args)
  File "./train_1d.py", line 190, in train
    loss_avg = model.train(data_gpu, learning_rate)
  File "/home/dmitryp/dpinchuk/dmvfn/CVPR2023-DMVFN/model/model_1d.py", line 92, in train
    loss_G.backward()
AttributeError: 'float' object has no attribute 'backward'

INFO: Sys.exc_info()[2]:
INFO: <traceback object at 0x7f898c802dc0>
